resources/dex/README.md

```diff
@@ -12,3 +12,49 @@ Currently, Dex uses a static user database and authenticates static users by its
 For the list of static Dex users and clients, as well as the information about the connectors that delegate authentication to external identity providers, see the [dex-config-map.yaml](templates/dex-config-map.yaml) file.
 
 Dex is exposed using the [Istio VirtualService](https://istio.io/docs/reference/config/networking/v1alpha3/virtual-service/) feature. Access Dex at `https://dex.kyma.local`.
+
+## Configuration
+
+This chart allows to provide configuration for Dex connectors and clients using Helm overrides mechanism.
+
+
+### Connectors
+
+Connectors can be configured using override named `connectors`.
+The value of the override must be a single string containing Dex connectors configuration in YAML format. See [Dex connectors documentation](https://github.com/dexidp/dex/tree/master/Documentation/connectors) for details.
+
+Example:
+```
+  connectors: |-
+    - type: saml
+      id: sci
+      name: SAP CI
+      config:
+        # Issuer for SAML Request
+        entityIssuer: dex.{{ .Values.global.domainName }}
+        ssoURL: https://{{ .Values.idp.tenant | default "someDefault" }}.{{ .Values.idp.domain | default "mytenant.mydomain.com" }}/saml2/idp/sso?sp=dex.{{ .Values.global.domainName }}
+        ca: {{ .Values.idp.caPath}}/ca.pem
+        redirectURI: https://dex.{{ .Values.global.domainName }}/callback
+        usernameAttr: mail
+        emailAttr: mail
+        groupsAttr: groups
+```
+
+### Clients
+Configuration for static clients is split in two parts: `staticClientsBase` and `staticClientsExtra`
```

Comment:
I'd make this section look a bit different. Focus on what the user can do. 

Configure Dex clients through the `oidc.staticClientsExtra` override. Pass the list of clients as a single string in the `yaml` format. 
>**WARNING:** The `oidc.(?)staticClientsBase` override defines the basic clients required by Kyma. Do not edit this override. 

This is an example of a valid client configuration string:

<string>

>**TIP:** you can use Go Template expressions in the override value. These expressions are resolved by Helm using the same set of overrides as configured for the entire chart.